code/src/constants/common-messages/en.json

```diff
@@ -1,4 +1,10 @@
 {
-  "login.label": "Login",
+  "errors.badRequest": "Invalid input data. Please check it and try again",
+  "errors.unauthorized": "User is not authorized to perform this action",
+  "errors.conflict": "Already exists",
+  "errors.somethingWentWrong": "Something went wrong. Please try again later",
+  "signIn.label": "Login",
+  "signIn.success": "You successfully signed in",
+  "signUp.success": "You successfully signed up",
   "logout.label": "Logout"
```

Comment:
if `logout` have `.label` try to write in one single approach. or can rename it

Reply:
I did not get your thought. Can you please explain what do you mean?

Reply:
> I did not get your thought. Can you please explain what do you mean?

Sure. For now we has existing `"logout.label"`
<img width="563" alt="image" src="https://github.com/idx-academy/spa-orders/assets/56305508/ac7fda9c-fd89-470b-811f-38cb2fcefcc7">

Just to write everything in one approach with `.label` or remove for existing

Reply:
I use label for buttons text